src/components/TextInput/TextInput.styles.js

```diff
@@ -0,0 +1,13 @@
+import {StyleSheet} from 'react-native'
+
+export const styles = StyleSheet.create({
+    textInputStyles:{
+        backgroundColor:'#fff',
+        borderColor:'#e5e5e5',
+        borderWidth:1,
+        borderRadius:8,
+        width:'100%',
+        marginBottom:20,
```

Comment:
This depends on a particular screen. you have to have a way to provide extra styles based on a particular page and situation. think about the component itself. on some pages, it will have 20, somewhere 50, etc.